### ISSUE TITLE
Allow modifier parens on @private?

### ISSUE THREAD

**Issue by zepumph** [phetsims/tasks#982]
@samreid @chrisklus @jessegreenberg @jonathanolson @zepumph @jbphet were involved in a discussion around the `@private` annotations and adding specifying parens afterwards, like in `RectangularButtonView.js`

```
    // @private (a11y) {PressListener}
    this._pressListener = buttonModel.createListener( {
```
That was added while converting the press listener from a `var` to a member variable (property). The annotation was specified because if it weren't for its a11y usage it wouldn't have to be on `this`. 

After discussing one thought was that for the most part "specificity" (in the parens) about where to use private vars increases the documentation load, and shouldn't be added there. We never really came to a consensus, there are two possibilities forward I think:

* Omit the ability to add annotations after `@private` (though `read-only` should be exempt). Then remove all usages of that in the project (I found 51 but many are `read-only`)
* Continue allowing them

**Comment by zepumph** [comment]
The above is a bad example, and I don't want to divert this issue. What if the above press listener was private and read only? 

```js
 // @private (read-only) {PressListener}
    this._pressListener = buttonModel.createListener( {
```

that looks reasonable to me, but maybe no?

**Comment by samreid** [comment]
Yes, we should keep the read-only modifier.

**Comment by zepumph** [comment]
Consensus during dev meeting, we decided that firstly, we reiterated that the modifier (like `(phet-io)`) is just that, it modifies the annotation, rather than expanding on it or explaining it. With that in mind we decided that for `@private` there is no need for modifiers, save `read-only` because it only applies to that file anyways. We also don't want to add documentation that needs to be kept in sync as you change code, like adding `// this property was added for a11y` doesn't make sense because though it was added for a11y, it could be now used however in that file, and we don't want to have to keep the doc in sync as we update the code.

Closing
